src/cmd/serve.rs

```diff
@@ -653,6 +653,29 @@ pub fn serve(
                         };
                         messages::report_elapsed_time(start);
                     }
+                    NoticeRemove(path) => {
+                        // Some editors, like those in the vi family, replace the file you're
```

Comment:
vim works for me, is it only vi having issues?

Reply:
On macOS, hot reloading works with vim/nvim for me. On Linux, neither work for me with `zola` built from master.